rpcs3/Emu/Cell/Modules/cellSaveData.cpp

```diff
@@ -2284,7 +2284,7 @@ error_code cellSaveDataUserFixedDelete(ppu_thread& ppu, u32 userId, PSetList set
 
 void cellSaveDataEnableOverlay(s32 enable)
 {
-	cellSaveData.error("cellSaveDataEnableOverlay(enable=%d)", enable);
+	cellSaveData.warning("cellSaveDataEnableOverlay(enable=%d)", enable);
```

Comment:
Didn't we have a Todo ?

Reply:
I don't think so.

Reply:
of course we do....

Reply:
You should have reworded the question because it asks if we had a "todo" log message for this function in the past.